shell/config/uiplugins.js

```diff
@@ -57,123 +50,132 @@ export const UI_PLUGIN_CHART_ANNOTATIONS = {
   EXTENSIONS_API: 'extensionsApiVersion',
   KUBE:           'kubeVersion',
-  HOST:           'host'
+  HOST:           'host',
 };
 
 export const EXTENSIONS_INCOMPATIBILITY_DATA = {
   UI: {
     type:           EXTENSIONS_INCOMPATIBILITY_TYPES.UI,
     cardMessageKey: 'plugins.incompatibleRancherVersion',
     tooltipKey:     'plugins.info.requiresRancherVersion',
-    mainHost:       UI_PLUGIN_HOST_APP
   },
   EXTENSIONS_API: {
     type:           EXTENSIONS_INCOMPATIBILITY_TYPES.EXTENSIONS_API,
     cardMessageKey: 'plugins.incompatibleUiExtensionsApiVersion',
     tooltipKey:     'plugins.info.requiresExtensionApiVersion',
-    mainHost:       UI_PLUGIN_HOST_APP
   },
   KUBE: {
     type:           EXTENSIONS_INCOMPATIBILITY_TYPES.KUBE,
     cardMessageKey: 'plugins.incompatibleKubeVersion',
     tooltipKey:     'plugins.info.requiresKubeVersion',
-    mainHost:       UI_PLUGIN_HOST_APP
   },
   HOST: {
     type:           EXTENSIONS_INCOMPATIBILITY_TYPES.HOST,
     cardMessageKey: 'plugins.incompatibleHost',
     tooltipKey:     'plugins.info.requiresHost',
-    mainHost:       UI_PLUGIN_HOST_APP
-  }
+    mainHost:       UI_PLUGIN_HOST_APP,
+  },
 };
 
 export function isUIPlugin(chart) {
-  return !!chart?.versions.find((v) => {
-    return v.annotations && v.annotations[UI_PLUGIN_ANNOTATION_NAME] === UI_PLUGIN_ANNOTATION_VALUE;
-  });
+  return !!chart?.versions.find((v) => v.annotations?.[UI_PLUGIN_ANNOTATION.NAME] === UI_PLUGIN_ANNOTATION.VALUE);
 }
 
 export function uiPluginHasAnnotation(chart, name, value) {
-  return !!chart?.versions.find((v) => {
-    return v.annotations && v.annotations[name] === value;
-  });
+  return !!chart?.versions.find((v) => v.annotations?.[name] === value);
 }
 
 /**
- * Get value of the annotation from teh latest version for a chart
+ * Get value of the annotation from the latest version for a chart
  */
 export function uiPluginAnnotation(chart, name) {
-  if (chart?.versions?.length > 0) {
-    return chart.versions[0].annotations?.[name];
-  }
-
-  return undefined;
+  return chart?.versions?.[0]?.annotations?.[name];
 }
 
 /**
- * Parse the rancher version string
+ * Parse the Rancher version string
  */
 function parseRancherVersion(v) {
-  let parsedRancherVersion = semver.coerce(v)?.version;
-  const splitArr = parsedRancherVersion.split('.');
+  let parsedVersion = semver.coerce(v)?.version;
+  const splitArr = parsedVersion?.split('.');
 
   // this is a scenario where we are on a "head" version of some sort... we can't infer the patch version from it
   // so we apply a big patch version number to make sure we follow through with the minor
   if (v.includes('-') && splitArr?.length === 3) {
-    parsedRancherVersion = `${ splitArr[0] }.${ splitArr[1] }.999`;
+    parsedVersion = `${ splitArr[0] }.${ splitArr[1] }.999`;
   }
 
-  return parsedRancherVersion;
+  return parsedVersion;
 }
 
-// i18n-uses plugins.error.generic, plugins.error.api, plugins.error.host
```

Comment:
This has been lost. This informs the process that checks strings are used... that these strings are used here. Looks like there still are, plus some new

Reply:
Will add it back 🙏

Reply:
Could find these, or the new entries, sure they were added?

Reply:
Sure thing. Just check the changes in `en-us.yaml` around line 4290

Reply:
as in the comment has been added back in and updated with the new strings.

it would help if i could type as well. `Couldn't find these, or the new entries, sure they were added?`